streamly.cabal

```diff
@@ -726,8 +730,11 @@ test-suite version-bounds
   type: exitcode-stdio-1.0
   main-is: version-bounds.hs
 
-executable FileSystem.Event
+test-suite FileSystem.Event
   import: test-options
-  main-is: Streamly/Test/FileSystem/Event.hs
-  if !flag(manual-tests)
-    buildable: False
+  type: exitcode-stdio-1.0
+  main-is: Streamly/Test/Internal/FileSystem/Event.hs
```

Comment:
We should move `Streamly/Test/Internal/FileSystem/Event.hs` to `Streamly/Test/FileSystem/Event.hs` and use the latter here, and remove the former. The CI tests are failing because the file `Streamly/Test/FileSystem/Event.hs` is present in the git tree but is not being used anywhere in cabal file.

Reply:
Also, we should move the old file `Streamly/Test/FileSystem/Event.hs` to `streamly-examples` repo, it can act as a simple example instead of a test.